tortoise/query_utils.py

```diff
@@ -78,6 +79,23 @@ def get_joins_for_related_field(
     return required_joins
 
 
+def expand_lookup_expression(root_model: type[Model], lookup_expression: str) -> Sequence[Field]:
+    field_names = lookup_expression.split("__")
+    fields: list[Field | RelationalField] = []
+    model = root_model
+    for field_name in field_names[:-1]:
+        if field_name not in model._meta.fetch_fields:
+            raise FieldError(f"{lookup_expression} not resolvable")
+        field = cast(RelationalField, model._meta.fields_map[field_name])
+        fields.append(field)
+        model = field.related_model
+    try:
+        fields.append(model._meta.fields_map[field_names[-1]])
```

Comment:
Cloud you leave some comment for why put the last element out of the for loop?

Reply:
updated!